File: offer-search-service/src/main/java/pl/allezon/offersearch/MainApp.java
package pl.allezon.offersearch;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import pl.allezon.offersearch.config.KafkaConsumerProperties;

@SpringBootApplication
@EnableConfigurationProperties(KafkaConsumerProperties.class)
public class MainApp {
    public static void main(String[] args) {
        SpringApplication.run(MainApp.class, args);
    }
}

File: offer-search-service/src/main/java/pl/allezon/offersearch/MainApp.kt
package pl.allezon.offersearch

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class MainApp {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(MainApp::class.java, *args)
        }
    }

}
